src/common/media.cpp

```diff
@@ -272,6 +274,11 @@ void media_loop(void) {
                     }
                     break;
                 }
+                while (len == MAX_CMD_SIZE && buffer[MAX_CMD_SIZE - 1] != '\n') {
+                    // Read until the end of the line
+                    f_gets(buffer, sizeof(buffer), &media_print_fil);
```

Comment:
/nitpick/ In the case that this `f_gets` call starts to fail, we might end up in an infinite cycle. Right?

Reply:
Good point, thanks